Add `lastUpdated` global and frontmatter options (#256)
withastro/starlight: docs/src/content/docs/reference/configuration.md

  content?: string;
}
```

> ### `showUpdateDate`

Reviewer: Small API detail: I think it would be nice if the config file and frontmatter options had the same name — I’d suggest `lastUpdated` for both.

It’s been bugging me that I ended up with `editLink` in config and `editUrl` in frontmatter — I might still make a breaking change at some point to align those 😅 

```suggestion
### `lastUpdated`
```

**type:** `boolean`  
**default:** `false`

> Define if the last update date should be visible in the page footer.

Reviewer 2: ```suggestion
Control whether the footer shows when the page was last updated.
```
I kind of like something like this! It makes it explicit that the display will ONLY happen in the footer or not, and doesn't duplicate "update date".

> By default, this feature relies on the Git history and may not be accurate on some deployment platforms performing [shallow clones](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). A page’s frontmatter can override this setting or the generated date using the [`lastUpdated`](/reference/frontmatter/#lastupdated) field.

Reviewer: ```suggestion
By default, this feature relies on your repository’s Git history and may not be accurate on some deployment platforms performing [shallow clones](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). A page can override this setting or the Git-based date using the [`lastUpdated` frontmatter field](/reference/frontmatter/#lastupdated).
```
